### ISSUE TITLE
i am not able submit form with post method using cy.request()

### ISSUE THREAD

**Issue by codd22** [cypress-io/cypress#4379]
### Current behavior:

using `cy.request()` i am not able to reset password.

I am able to verify reset password form but not able to submit it with data:
email, password, confirm password.

### Desired behavior:

should able to reset password after passing email, password, confirm password data as parameter in `cy.request()`.

### Steps to reproduce: (app code and test code)

```js
cy.request({
        method: 'POST',
        url: href, 
        form: true, 
        body: {
          Email,
          Password,
          ConfirmPassword
        },
      })
```

### Versions
cypress: 2.1.0

**Comment by jennifer-shehane** [comment]
@codd22 Could you please run this with Cypress version 3.3.1. Version 2.1.0 is quite an old version at this point. Your issue may already be resolved. 

If not resolved in 3.3.1, we'll need a reproducible example or a screenshot of what Cypress is sending incorrectly that is different from outside of cypress.

**Comment by jennifer-shehane** [comment]
Unfortunately we have to close this issue due to inactivity. Please comment if there is new information to provide concerning the original issue and we can reopen.
